pkg/progress/progressui/display.go

```diff
@@ -805,14 +793,14 @@ func (disp *display) print(d displayInfo, width, height int, all bool) {
 
 	statusStr := ""
 	if d.countCompleted > 0 && d.countCompleted == d.countTotal && all {
-		statusStr = "✅ (finished)"
+		statusStr = "FINISHED"
 	}
 
 	fmt.Fprint(disp.c, aec.Hide)
 	defer fmt.Fprint(disp.c, aec.Show)
 
-	out := fmt.Sprintf("[+] 🐋 %s %.1fs (%d/%d) %s", disp.phase, time.Since(d.startTime).Seconds(), d.countCompleted, d.countTotal, statusStr)
-	// out = align(out, "", width)
+	out := fmt.Sprintf("[+] %s %.1fs (%d/%d) %s", disp.phase, time.Since(d.startTime).Seconds(), d.countCompleted, d.countTotal, statusStr)
```

Comment:
Does this fix the indent problem?

Reply:
Yep.

Reply:
But the main reason is to fix one flaky test caused by the function.